.buildkite/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   env: &xcode_image
-    IMAGE_ID: xcode-12.5.1
+    IMAGE_ID: xcode-13.4.1
```

Comment:
I also had to bump the CI image used here because our `xcode-12.5.1` image [didn't have ruby 2.7.4 installed on it](https://github.com/Automattic/buildkite-ci/blob/trunk/src/macos-vms/manifests/xcode-12.5.1.json#L7-L9). It was about time we update the CI image used to a more recent one anyway.